### PR TITLE
CW0001-T0001.3 : Implement the server & client side logic

### PR DESCRIPTION
**Pull request by codewarriors0002**
This web service is implemented to give stock prices of all the turns, to the client, But it give only the dummy values as the response.

### CODE REVIEW

File: src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@RequestMapping("/stockMarket")
+public class StockMarketSharesDetailsController {
+	
+	@GetMapping("/getMarketPrices")
+	public Market getStokMarketSharsPrices() {
+		List<Company> companies= new ArrayList<Company>();
+		Market market=new Market();
+		
+		market.setGameTime(60);// game time is 0s
+		market.setTurnTime(6);// companies stock price will change after 6s
```
Comment (codewarriors0002) [inline src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java]: is this time enough??

File: src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+
+import models.Company;
+import models.Market;
+import models.Turn;
```
Comment (codewarriors0004) [inline src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java]: package model name has to be changed to com.codewarriors.models

File: src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
+package com.codewarriors.controllers;
```
Comment (codewarriors0001) [inline src/main/java/com/codewarriors/controllers/StockMarketSharesDetailsController.java]: code was fine

**Comment by codewarriors0006** [comment]
shall we increase gametime up to 1 min ? @codewarriors0001 @codewarriors0002 @codewarriors0003 @codewarriors0004 @codewarriors0005

**Pull request closed without merging**